editor/src/clj/editor/gui.clj

```diff
@@ -2574,6 +2574,9 @@
   (output texture-resource-names GuiResourceNames :cached (g/fnk [names] (into (sorted-set) names)))
   (input build-errors g/Any :array)
   (output build-errors g/Any (gu/passthrough build-errors))
+  (input texture-page-counts g/Any :array)
+  (output texture-page-counts g/Any :cached (g/fnk [texture-page-counts]
+                                              (into {} cat texture-page-counts)))
```

Comment:
Now we aggregate `texture-page-counts` from all textures on a `TexturesNode`, and have a single non-array connection from here to `GuiSceneNode`.